IncidentFields/incident-phishingsubtype.json

```diff
@@ -0,0 +1,51 @@
+{
+    "id": "incident_subtype",
+    "version": -1,
+    "modified": "2019-08-21T13:40:48.682176491Z",
+    "commitMessage": "",
+    "shouldPublish": false,
+    "shouldCommit": false,
+    "shouldPush": false,
+    "name": "Sub-type",
+    "prevName": "Sub-type",
+    "ownerOnly": false,
+    "placeholder": "",
+    "description": "Sub-type of phishing incident.",

```

Comment:
Phishing incident sub-type. Can be "Legitimate", "Spam", "Malicious", "Malware", "Credential Harvesting", or "Scam".